#!/usr/bin/python3


def raise_exception_msg(message=""):
    """Raise a NameError message."""
    raise NameError(message)
